dashboard/app/models/rubric.rb

```diff
@@ -9,4 +9,7 @@
 #  updated_at :datetime         not null
 #
 class Rubric < ApplicationRecord
+  has_many :learning_goals, -> {order(:position)}, dependent: :destroy
+  belongs_to :level
+  belongs_to :lesson
```

Comment:
One thing I should've called out -- as written, both a lesson and a level are required. It's easy to mark them as optional but I want to check what feels right here. My working assumption has been that both are needed but I might be wrong there.

Reply:
The core reason why both would be required, to my understanding going in, would be because there could be more than one per lesson. but I guess we want both... since a level is absolutely required to be the point of assessment and then a lesson is loosely required for the purposes of lookup and showing it on the lesson page more easily.

Reply:
Yeah, that was my thinking as well. I'll leave it as-is and as we work and potentially find reasons to adjust this, we can reconsider